lib/datasource.js

```diff
@@ -2097,7 +2097,11 @@ DataSource.prototype.columnNames = function(modelName) {
 };
 
 /**
- * Find the ID column name
+ * @deprecated
+ * Function to get the id column name of the target model
+ *
+ * Use `SqlConnector.prototype.column(ModelDefinition.prototype.idName())` instead
```

Comment:
Shouldn't this be:

`Connector.prototype.idMapping` ?

Reply:
Reworded